commit b661c34ac60daeb13f8fd52ee06f5101583334cd (refs/changes/02/4702/4)
Author: Dan McDonald <danmcd@joyent.com>
Date:   2018-08-29T20:08:52-04:00 (1 year, 1 month ago)
    
    OS-7141 Overlay device-creation weirdness contributes to varpd boot failures
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Reviewed by: Jason King <jbk@joyent.com>

diff --git a/overlay/generic/etc/name_to_major b/overlay/generic/etc/name_to_major
index cfab8bff..26eaeddb 100644
--- a/overlay/generic/etc/name_to_major
+++ b/overlay/generic/etc/name_to_major
@@ -193,6 +193,7 @@ ixgbe 192
 bpf 193
 lsimega 194
 vioif 195
+overlay 196
 mpt_sas 198
 mr_sas 199
 mega_sas 200
